packages/release/activate-configured.service

```
@@ -0,0 +1,12 @@
+[Unit]
+Description=Isolate configured.target
+After=preconfigured.target
```

We might want also want
```suggestion
After=preconfigured.target
Requires=preconfigured.target
```